docs/python/formatting.md

```diff
@@ -36,6 +61,20 @@ For example, to set Black Formatter as the default formatter, add the following
   }
 ```
 
+In order to set a formatter extension as an import sorter, you can set your preference under `"editor.codeActionsOnSave"` in your User `settings.json` file, under a `[python]` scope.
+
+For example, to set Ruff as your preferred import sorter, add the following setting to your User `settings.json` file:
```

Comment:
These settings can wither go to user or in workspace settings. From the doc it feels like it can only be in user.

Reply:
I actually didn't realize we could do this in Workspace settings as well. If we review the README.md files for various tools it might be worth spelling out User and/or Workspace settings when applicable. I know User vs Workspace settings is confusing for users. I noticed the isort extension just says "settings". 

Pushed these changes.